dmci/distributors/pycsw_dist.py

```diff
@@ -119,16 +147,16 @@ def _delete(self):
             'xsi:schemaLocation="http://www.opengis.net/cat/csw/2.0.2 '
             'http://schemas.opengis.net/csw/2.0.2/CSW-publication.xsd" '
             'service="CSW" version="2.0.2">'
-            '   <csw:Delete>'
-            '       <csw:Constraint version="1.1.0">'
-            '           <ogc:Filter>'
-            '               <ogc:PropertyIsEqualTo>'
-            '                   <ogc:PropertyName>apiso:Identifier</ogc:PropertyName>'
-            '                   <ogc:Literal>%s</ogc:Literal>'
-            '               </ogc:PropertyIsEqualTo>'
-            '           </ogc:Filter>'
-            '       </csw:Constraint>'
-            '   </csw:Delete>'
+            '  <csw:Delete>'
```

Comment:
should we also define this as binary (i.e., like `b'some text'`)?

Reply:
requests.post handles both str and bytes, so I think we can keep it like this (To not have to encode _metadata_id, as it is str)